### PR TITLE
LPS-138216 Change Object Name to Object Label in the Portlet and Object Admin

### PR DESCRIPTION
**Pull request by mlan0**

### CODE REVIEW

**Comment by mlan0** [comment]
ci:test:relevant

**Comment by mlan0** [comment]
ci:test:stop

**Comment by mlan0** [comment]
ci:stop

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/LanguageUtils.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * @author Matheus Almeida
+ */
+public class LanguageUtils {
+
+	public static Map<String, String> getLanguageIdMap(
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/util/LanguageUtils.java]: Hey @mlan0 ,

Would you mind to move this method to the class com.liferay.object.util.LocalizedMapUtil?
Another option is take a look on this class (com.liferay.portal.vulcan.util.LocalizedMapUtil) checking if some existing method can help you with. 

Thanks!

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/application/list/ObjectEntriesPanelApp.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public String getKey() {
 
 	@Override
 	public String getLabel(Locale locale) {
-		return _objectDefinition.getShortName();
+		return _objectDefinition.getLabel(locale);
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/application/list/ObjectEntriesPanelApp.java]: The use of PluralLabel is better here

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/application/list/ObjectEntriesPanelApp.java (hunk 2 above)
Comment (mlan0) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/application/list/ObjectEntriesPanelApp.java]: I changed it!

**Comment by mlan0** [comment]
ci:test:sf

**Comment by mlan0** [comment]
ci:test:relevant

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Fixed Poshi tests: https://github.com/liferay-objects/liferay-portal/pull/127

**Comment by rodrigocunhaa** [comment]
ci:close

**Pull request closed without merging**